DominantColor/Shared/DominantColors.swift

```diff
@@ -161,19 +161,19 @@ private func distanceForAccuracy(accuracy: GroupingAccuracy) -> (INVector3, INVe
     case .Low:
         return CIE76SquaredColorDifference
     case .Medium:
-        return CIE94SquaredColorDifference()
+        return { CIE94SquaredColorDifference()(colors: $0) }
```

Comment:
I also thought the same thing. I don't recall any explicit changes that would affect this, but it is causing a compile error for me.

The basic issue is that the function signature for the three internal functions are different, and this creates the compiler error. CIE76SquaredColorDifference defines a signature with two ```INVector3``` as parameters, and the other two define a signature with a single tuple parameter made up of two ```INVector3``` objects.

```
DominantColor/DominantColor/Shared/DominantColors.swift:166:16: Function signature '(colors: (INVector3, INVector3)) -> Float' is not compatible with expected type '(INVector3, INVector3) -> Float'
```

The issue could be also be fixed by altering the function signature for ```CIE94SquaredColorDifference``` and ```CIE2000SquaredColorDifference``` to expect two distinct parameters rather than a single tuple.

I can easily alter this to change the function signatures rather than the calling pattern if desired.